platforms/web/lib/useListeners/utils.ts

```diff
@@ -42,5 +42,6 @@ export function mapToAllActionStates(
                 key.substring(1)) as ActionTypes
         ] = value.toLowerCase() as ActionState;
     }
+    console.log('actionStatesMap', actionStatesMap);
```

Comment:
This is probably just leftover from integration work

Reply:
Good catch !